code/modules/hydroponics/grown/towercap.dm

```diff
@@ -162,6 +165,7 @@
 
 /obj/structure/bonfire/Crossed(atom/movable/AM, oldloc)
 	if(burning)
+		add_attack_logs(src, AM, "burned by a bonfire (Lit by [lighter])", ATKLOG_FEW)
```

Comment:
`ATKLOG_FEW` means admins with Few logs turned on (aka they see few logs) get them. You prob want `ATKLOG_ALMOSTALL` for this one.

Reply:
Definitely not ATKLOG_FEW. That's reserved for things like attacks on SSD players, bombs, etc which are very possibly rule violations.
I'd probably go with ATKLOG_MOST. The major difference between ATKLOG_ALMOSTALL and ATKLOG_MOST is that ATKLOG_MOST hides NPC v player combat. You could argue that a bonfire should be treated like an NPC (its player-created, but not player-controlled).
(semi-joking) perhaps creating bonfires should be logged at a higher level than the burns they inflict, though. Perhaps creating bonfires in key areas, like the escape shuttle, or the Escape room, should even be logged as ATKLOG_FEW, since its likely griefing and will affect many players.